src/AppBundle/DataFixtures/ORM/LoadAdherentData.php

```diff
@@ -148,6 +166,14 @@ public function load(ObjectManager $manager)
             'address' => PostAddress::createFrenchAddress('30 Boulevard Louis Guichoux', '13003-13203'),
         ]);
 
+        $committee3 = $committeeFactory->createFromArray([
```

Comment:
This committee is not activated btw so it shouldn't be selected in the committees list. So make sure you're retrieving only approved committees when you fetch them from the db.